src/components/pages/document/show/sidebar/table-scientific-names.tsx

```diff
@@ -0,0 +1,83 @@
+// eslint-disable-next-line simple-import-sort/imports
```

Comment:
please run autofix from vscode instead of disabling eslint rule

Reply:
Correction added.